crates/wast/src/component/resolve.rs

```diff
@@ -383,9 +383,16 @@ impl<'a> Resolver<'a> {
             CanonicalFuncKind::ResourceDrop(info) => {
                 self.resolve_ns(&mut info.ty, Ns::Type)?;
             }
-            CanonicalFuncKind::ThreadSpawn(info) => {
+            CanonicalFuncKind::ThreadSpawnRef(info) => {
                 self.resolve_ns(&mut info.ty, Ns::CoreType)?;
             }
+            CanonicalFuncKind::ThreadSpawnIndirect(info) => {
+                self.core_item_ref(&mut info.table)?;
+                // Eventually this should resolve the specific type associated
+                // with this canonical function, e.g.,
+                // `self.resolve_type_use(&mut info.ty)?;` (TODO: spawn indirect
+                // types).
```

Comment:
Adding a comment to flag this. This looks though like it's a TODO at the spec-level, right? (as opposed to just exclusively the implementation here)